src/checkoutservice/main.go

```diff
@@ -168,11 +187,14 @@ func (cs *checkoutService) PlaceOrder(ctx context.Context, req *pb.PlaceOrderReq
 		return nil, status.Errorf(codes.Internal, "failed to charge card: %+v", err)
 	}
 	log.Infof("payment went through (transaction_id: %s)", txID)
+	span.AddEvent("charged", trace.WithAttributes(attribute.String("app.order.transaction.id", txID)))
 
 	shippingTrackingID, err := cs.shipOrder(ctx, req.Address, prep.cartItems)
 	if err != nil {
 		return nil, status.Errorf(codes.Unavailable, "shipping error: %+v", err)
 	}
+	shippingTrackingAttribute := attribute.String("app.order.tracking.id", shippingTrackingID)
```

Comment:
Can we use the same attribute name as the one used in the emailservice?
```suggestion
	shippingTrackingAttribute := attribute.String("app.shipping.tracking.id", shippingTrackingID)
```